composer/datasets/ade20k_hparams.py

```diff
@@ -119,10 +112,9 @@ class StreamingADE20kHparams(DatasetHparams):
         ignore_background (bool): if true, ignore the background class when calculating the training loss.
             Default: ``true``.
     """
-
     version: int = hp.optional('Version of streaming (1 or 2)', default=1)
     remote: str = hp.optional('Remote directory (S3 or local filesystem) where dataset is stored',
-                              default='s3://mosaicml-internal-dataset-ade20k/mds/1/')
+                              default='s3://mosaicml-internal-dataset-ade20k/mds/2/')
```

Comment:
is this changing the streaming remote version default? It looks like the default _version_  hasn't been does this need to match?

Reply:
PR: https://github.com/mosaicml/composer/pull/1616 changes the default version to 2. Rebasing this PR to `dev` branch should fix the issue.

Reply:
Rebasing 👍